gravitee-apim-gateway/gravitee-apim-gateway-services/gravitee-apim-gateway-services-sync/src/main/java/io/gravitee/gateway/services/sync/kubernetes/KubernetesSyncService.java

```diff
@@ -59,6 +61,9 @@ public class KubernetesSyncService extends AbstractService<KubernetesSyncService
     private final ObjectMapper mapper;
     private Disposable disposable;
 
+    @Value("${services.sync.kubernetes.namespace:#{null}}")
```

Comment:
Could it be à list of namespace ?

Reply:
Yes, a comma separated list of strings like "ns1,ns2,ns3"